src/CheckoutSdk/Payments/AlternativePaymentSource.cs

```diff
@@ -9,9 +9,15 @@ public class AlternativePaymentSource : Dictionary<string, string>, IRequestSour
     {
         private const string TypeField = "type";
 
+        /// <summary>
+        /// Creates a new default <see cref="AlternativePaymentSource"/> instance.
+        /// </summary>
+        public AlternativePaymentSource() { }
```

Comment:
Why was this added? It should not be possible to create an `AlternativePaymentSource` without a `type`